src/Sylius/Bundle/CoreBundle/Resources/config/services/payment_request/offline.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <defaults public="true" />
+
+        <service id="sylius.payment_request.command_provider.offline" class="Sylius\Bundle\CoreBundle\PaymentRequest\CommandProvider\PaymentRequestTypeCommandProvider">
+            <argument type="tagged_locator" tag="sylius.payment_request.command_provider.offline" index-by="type" />
+            <tag name="sylius.payment_request.command_provider" factory="offline" priority="-100" />
```

Comment:
Surely we don't, we have to keep in mind that a `PaymentMethod` is an artefact of a gateway factory.
Factory is the right word I think here since it is something that create the `PaymentMethod`.

Reply:
Renamed.